Documentation/tpv13.rst

```diff
@@ -3,7 +3,9 @@
 SCEC TPV13
 ==========
 
-TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. TPV13 is using non-associative Drucker-Prager plasticity with yielding in shear. The material is characterized by six constitutive parameters:
+TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. To run TPV13, it requires to recompile SeisSol with "plasticity = 'yes'" in the configuration python script. 
```

Comment:
SeisSol must be compiled with "-DPLASTICITY=ON".

Reply:
.. and the new SeisSol CMake compilation does not use the configuration python script anymore.
Can we just use more general terms? in example: 
To run TPV13, it requires to recompile SeisSol with the plasticity option activated in the configuration file.